comfy/model_management.py

```diff
@@ -85,10 +86,17 @@
 except:
     pass
 
+try:
+    import intel_extension_for_pytorch
```

Comment:
This does need to be `import intel_extension_for_pytorch as ipex` thanks to some annoying name aliasing side-effects they rely on... causes weird intermittent issues otherwise. *sigh*

Reply:
> This does need to be `import intel_extension_for_pytorch as ipex` thanks to some annoying name aliasing side-effects they rely on... causes weird intermittent issues otherwise. _sigh_

already fixed. thx!